### DISCUSSION TITLE
auto_https disable_redirects

### DISCUSSION THREAD

**Discussion by pomology** [eriksjolund/podman-caddy-socket-activation#24] [General]
What is the reason for the "auto_https disable_redirects" in the Caddyfile?  

Thanks!

**Comment by eriksjolund** [comment]
Thanks, for pointing it out. I removed those lines:

* https://github.com/eriksjolund/podman-caddy-socket-activation/pull/25

**Comment by qrhfz** [comment]
why did it come back in 5593bfff5c9bc8d65c6776ac6ba7f62ec4b36f1c ?

**Reply by eriksjolund** [reply]
Ooops, that was not my intention.

I created
https://github.com/eriksjolund/podman-caddy-socket-activation/pull/50

**Comment by eriksjolund** [comment]
The line  
```
auto_https disable_redirects
```
is needed.

See the explanation by @betelgeuse  in 

https://github.com/eriksjolund/podman-caddy-socket-activation/commit/261cc556a7446ca65e76a89eafd6c5fb2a9ce24a

that was merged in

* https://github.com/eriksjolund/podman-caddy-socket-activation/pull/70
